src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -66,13 +98,28 @@ def get(self, nqn: str):
         )
         @empty_response
         @handle_nvmeof_error
-        def create(self, nqn: str, enable_ha: bool, max_namespaces: int = 256):
-            return NVMeoFClient().stub.create_subsystem(
+        def create(self, nqn: str, enable_ha: bool,
+                   max_namespaces: int = 256, initiators: str = None):
+            subsystem_response = NVMeoFClient().stub.create_subsystem(
                 NVMeoFClient.pb2.create_subsystem_req(
                     subsystem_nqn=nqn, max_namespaces=max_namespaces, enable_ha=enable_ha
                 )
             )
 
+            if subsystem_response.status != 0:
+                return subsystem_response
```

Comment:
This should be done by the handle_nvmeof_error decorator.

Reply:
yes, but here we have requests followed up so checking here if all good to proceed or not.
I just copied this way from what we have been doing at rest of the places like iops api for namespaces